NBitcoin/Key.cs

```diff
@@ -407,6 +408,18 @@ public BitcoinSecret GetWif(Network network)
 			return new BitcoinSecret(this, network);
 		}
 
+		public byte[] GetSeedBytes()
+		{
+			AssertNotDiposed();
+			byte[] bytes = Enumerable.Repeat((byte)0x00, KEY_SIZE).ToArray();

```

Comment:
`new byte[32]` is  clearly overrated

Reply:
oops! didn't realize that the bytes would be overriden later anyway, was just monkey-translating F# code to C# code from somewhere else